challenges/1/samples.csv

```diff
@@ -0,0 +1,23 @@
+sample_id,cross,role,n_reads,median_cov,mean_cov,sex,accession,ftp_path
```

Comment:
What is the ``n_reads`` column, and is it correct?

Reply:
taken directly from `/kwiat/vector/ag1000g/release/phase1.AR3.1/samples/cross.samples.meta.txt`

Agree it looks strange- will see if I can find where that came from.

Reply:
AD0231-C should have 45176208 reads. I think this error must have crept in when trying to use the `pandas` round functions in `to_csv`. Let me fix this!